.changeset/dirty-cycles-lick.md

```diff
@@ -0,0 +1,36 @@
+---
+'astro': minor
+---
+
+Add `astro/type-utils` entrypoint. These utilities can be used for common prop type patterns.
```

Comment:
Wondering if `type-utils` is the best name for this? I understand why these can't come from the `astro` global, but a simpler `astro/types` may be more flexible.

Reply:
Yeah I agree with that! `astro/types` seems better.